content/guides/concepts/hashes.md

```diff
@@ -19,24 +19,41 @@ would be represented by **SHA-1** as:
 0x7B502C3A1F48C8609AE212CDFB639DEE39673F5E
 ```
 
-However, exactly the same input generates the following output by **SHA-256**:
+However, the exact same input generates the following output using **SHA-256**:
 
 ```
 0x64EC88CA00B268E5BA1A35678A1B5316D212F4F366B2477232534A8AECA37F3C
 ```
 
-Notice that the second hash is longer than the first one. This is because SHA-1 creates a 160 bit hash, while SHA-256 creates a 256 bit hash. Also, the prepended `0x` is just and indicator that tells us that the following digest is represented as a hexadecimal number.
+Notice that the second hash is longer than the first one. This is because SHA-1 creates a 160 bit hash, while SHA-256 creates a 256 bit hash. Also, the prepended `0x` is just an indicator that tells us that the following hash is represented as a base 16 (or hexadecimal) number.
 
 Hashes can be represented in different bases (`base2`, `base16`, `base32`, etc.). In fact, IPFS makes use of that as part of its [Content Identifiers](cid.md) and supports mulitiple base representations at the same time, using the [Multibase](https://github.com/multiformats/multibase) protocol.
 
+For example, the SHA-256 hash of "Hello World" from above can be represented as base 32 as:
+
+```
+mtwirsqawjuoloq2gvtyug2tc3jbf5htm2zeo4rsknfiv3fdp46a
+```
+
+An easy way to play around with these in JavaScript is to use the [multibase package](https://github.com/multiformats/js-multibase):
+
+```
+const crypto = require('crypto');
+const multibase = require('multibase');
+
+const hashBytes = crypto.createHash('sha256').update('Hello world').digest();
+multibase.encode('base32', hashBytes).toString();
+```
+
+Just keep in mind that this will output a base 32 string including the multibase prefix.
```

Comment:
For someone new to hashes reading this document, I think this line is going to make no sense. If we keep this code sample, this line definitely need some rewording to explain what the multibase prefix is :\

Reply:
(But to be clear, I think it would be better to remove the code sample entirely. I’m just open to keeping it if you disagree.)